### DISCUSSION TITLE
How to play large video? 5400 X 1920

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#1964] [Q&A]
Can the latest version of air SDK play large-size videos?

I tried playing a 5400x1920 video, but nothing was displayed.

**Comment by itlancer** [comment]
Related issues:
https://github.com/airsdk/Adobe-Runtime-Support/issues/89
https://github.com/airsdk/Adobe-Runtime-Support/issues/196

**Reply by itlancer** [reply]
@Jason0247 
For my tests videos with resolution more than 4K could be played with latest AIR 33.1.1.929 and 50.0.0.1.

**Comment by Jason0247** [comment]
@itlancer What is the highest resolution you tested? Is the encoding H.264?

**Reply by itlancer** [reply]
@Jason0247, for different platforms I tested many different resolutions. I will describe here maximum that I have tested. In all cases MP4 H.264 videos without specific encoding settings or without high profiles. High profiles could cause some errors https://github.com/airsdk/Adobe-Runtime-Support/issues/139
All tests with latest AIR 33.1.1.929 version using `StageVideo`. `Video` and `VideoTexture` could have limits with high resolutions with some uses cases/devices.

**Windows 10 64-bit (also you should use 64-bit AIR application!)**: 7540x3840@30
But with such 8K videos playback has extremely low performance https://github.com/airsdk/Adobe-Runtime-Support/issues/2162
With resolution above 8K (I think 8192 pixels per side) it doesn't work: https://github.com/airsdk/Adobe-Runtime-Support/issues/2163

**macOS**: 7540x3840@30 and 11520x4320@30
But with such 8K (and more) videos playback has extremely low performance https://github.com/airsdk/Adobe-Runtime-Support/issues/2162

**Linux**: video playback doesn't work right now https://github.com/airsdk/Adobe-Runtime-Support/issues/1984

**Android**: 5760x1080@30 and 5120x2160@30
But it could depends on device. Some low-performance devices doesn't support even 4K videos playback.

**iOS**: 7540x3840@30 and 11520x4320@30
But it could depends on device. Some old devices doesn't support such resolutions.

**Comment by Jason0247** [comment]
@itlancer Indeed, 7540x3840 can play, but it is very slow.
